deploy/kubernetes/jupyterhub-configs.yaml

```diff
@@ -7,6 +7,11 @@ data:
     import os,sys
     from oauthenticator.generic import LocalGenericOAuthenticator
 
+    class CustomLocalOAuthenticator(LocalGenericOAuthenticator):
+    def normalize_username(self, username):
+	    hash = hashlib.sha1()
+	    hash.update(username.encode('utf-8'))
+	    return hash.hexdigest()[:32]
```

Comment:
Will this username hash be propagated to the rest of JupyterHub that relies on username? (ie. url, pvc)